File: google_shipping_helpline/lib/ui/screens/home_screen.dart
import 'package:flutter/material.dart';

class HomeScreen extends StatelessWidget {
  const HomeScreen({super.key});

  @override
  Widget build(BuildContext context) {
    return Scaffold(
      appBar: AppBar(
        title: const Text('Transport Planner'),
      ),
      body: Center(
        child: Column(
          mainAxisAlignment: MainAxisAlignment.center,
          children: [
            ElevatedButton(
              onPressed: () {
                Navigator.pushNamed(context, '/routeComparison');
              },
              child: const Text('Route Comparison'),
            ),
            ElevatedButton(
              onPressed: () {
                Navigator.pushNamed(context, '/optimizationEngine');
              },
              child: const Text('Optimization Engine'),
            ),
            ElevatedButton(
              onPressed: () {
                Navigator.pushNamed(context, '/disruptionModeling');
              },
              child: const Text('Disruption Modeling'),
            ),
            ElevatedButton(
              onPressed: () {
                Navigator.pushNamed(context, '/transportCost');
              },
              child: const Text('Transport Cost Benchmarks'),
            ),
          ],
        ),
      ),
    );
  }
}

File: calm-red-almond/calm-red-almond/screens/HomeScreen.js
import React from 'react';
import { View, Text, Button } from 'react-native';

const HomeScreen = ({ navigation }) => (
  <View>
    <Text>Welcome to Google Maps for Shipping</Text>
    <Button title="Compare Routes" onPress={() => navigation.navigate('Route Comparison')} />
    <Button title="Optimize" onPress={() => navigation.navigate('Optimization Engine')} />
    <Button title="Disruption Modeling" onPress={() => navigation.navigate('Disruption Modeling')} />
    <Button title="Transport Cost" onPress={() => navigation.navigate('Transport Cost')} />
  </View>
);

export default HomeScreen;